component/core/src/main/java/org/exoplatform/social/core/listeners/SocialUserProfileEventListenerImpl.java

```diff
@@ -109,9 +109,9 @@ private void updateProfilePropertySettings(String propertyName, ProfilePropertyS
       ProfilePropertySetting profilePropertySetting = new ProfilePropertySetting();
       profilePropertySetting.setPropertyName(propertyName);
       profilePropertySetting.setMultiValued(false);
-      profilePropertySetting.setActive(true);
+      profilePropertySetting.setActive(false);
       profilePropertySetting.setEditable(false);
-      profilePropertySetting.setVisible(true);
+      profilePropertySetting.setVisible(false);
```

Comment:
Does this modification will create initial properties not active and not visible ?

Or only added properties (added in interface or programmatically like OIDC properties) ?

Reply:
@rdenarie This will set just added properties as inactive and invisible, to let the administrator decide whether he should activate them or not.
initial properties are activated and set visible using the XML configuration.